moveit_ros/planning/planning_scene_monitor/src/current_state_monitor.cpp

```diff
@@ -274,7 +274,13 @@ bool planning_scene_monitor::CurrentStateMonitor::waitForCurrentState(const ros:
     state_update_condition_.wait_for(lock, boost::chrono::nanoseconds((timeout - elapsed).toNSec()));
     elapsed = ros::WallTime::now() - start;
     if (elapsed > timeout)
+    {
+      ROS_INFO_STREAM("More than " << elapsed.toSec() << " seconds has passed to update robot state (joint angles)."
+                                   << "(longer than the allowed " << wait_time << " seconds)."
+                                   << " Synchronize the clocks between systems and check the connection if your are "
+                                      "running ros across multiple machines.");
```

Comment:
Suggestion: rearrange this a bit. Perhaps something like:

> If you are running ROS across multiple machines, check their connections and make sure time is properly synchronized on all of them

Right now it reads a bit like "synchronize the clocks" is something that is always needed, which could potentially confuse users that are not using multiple machines ("clocks? Which clocks?").